joy_node:
  ros__parameters:
    device_id: 0
    deadzone: 0.05
    autorepeat_rate: 20.0

teleop_node:
  ros__parameters:
    axis_linear:
      x: 1 # CHANGE
    scale_linear:
      x: 0.5
    scale_linear_turbo:
      x: 1.0

    axis_angular:
      yaw: 0 # CHANGE
    scale_angular:
      yaw: 0.5
    scale_angular_turbo:  
      yaw: 1.0

    enable_button: 6
    enable_turbo_button: 7

    require_enable_button: false

linear_position_pub:
  ros__parameters:
    min_value: 0.01
    max_value: 0.13
    axis_index: 4


    
    
